package.json

```diff
@@ -128,14 +128,13 @@
     "eslint-plugin-prettier": "3.1.2",
     "eslint-plugin-react": "7.18.0",
     "eslint-plugin-react-hooks": "2.3.0",
-    "eslint-plugin-testing-library": "1.4.1",
+    "eslint-plugin-testing-library": "1.5.0",
```

Comment:
I forgot to do it last time, we need to explicitly enable `prefer-expect-query-by` now, as it does not come enabled by default anymore: https://github.com/testing-library/eslint-plugin-testing-library/releases/tag/v1.4.1

More on the reasons here: https://github.com/testing-library/eslint-plugin-testing-library/issues/59

Reply:
FYI: we're replacing that rule with a new one, targeted only for cases when you want to assert that an element is not in the document.

https://github.com/testing-library/eslint-plugin-testing-library/pull/65

Reply:
A good to now. Thanks for sharing.

Reply:
Ah, interesting. Read up on the background now. Makes sense what's in store there :)